src/Components/Typography/styles.ts

```diff
@@ -0,0 +1,144 @@
+            `;
+      case TypographyTypes.HEADING_MEDIUM_BOLD:
+        return `  
+            font-size: 18px;
+            font-weight: bold;
+            line-height: 27px;
+            `;
+      case TypographyTypes.HEADING_MEDIUM_MEDIUM:
+        return `  
+            font-size: 18px;
+            font-weight: 500;
+            line-height: 27px;
+            `;
+      case TypographyTypes.HEADING_MEDIUM_REGULAR:
+        return `  
+            font-size: 18px;
+            font-weight: normal;
+            line-height: 27px;
+            `;
+      case TypographyTypes.SUBHEADING_BOLD:
+        return `  
+            font-size: 16px;
+            font-weight: bold;
+            line-height: 24px;
+            `;
+      case TypographyTypes.SUBHEADING_MEDIUM:
+        return `  
+            font-size: 16px;
+            font-weight: 500;
+            line-height: 24px;
+            `;
+      case TypographyTypes.SUBHEADING_REGULAR:
+        return `  
+            font-size: 16px;
+            font-weight: normal;
+            line-height: 24px;
+            `;
+      case TypographyTypes.BODY_BOLD:
+        return `  
+            font-size: 14px;
+            font-weight: bold;
+            line-height: 22px;
+            `;
+      case TypographyTypes.BODY_MEDIUM:
+        return `  
+            font-size: 14px;
+            font-weight: 500;
+            line-height: 22px;
+            `;
+      case TypographyTypes.BODY_REGULAR:
+        return `  
+            font-size: 14px;
+            font-weight: normal;
+            line-height: 22px;
+            `;
+      case TypographyTypes.CAPTION_BOLD:
+        return `  
+            font-size: 12px;
+            font-weight: bold;
+            line-height: 19px;
+            `;
+      case TypographyTypes.CAPTION_MEDIUM:
+        return `  
+            font-size: 12px;
+            font-weight: 500;
+            line-height: 19px;
+            `;
+      case TypographyTypes.CAPTION_REGULAR:
+        return `  
+            font-size: 12px;
+            font-weight: normal;
+            line-height: 19px;
+            `;
+      case TypographyTypes.XSMALL_BOLD:
+        return `  
+            font-size: 11px;
+            font-weight: bold;
+            line-height: 16px;
+            `;
+      case TypographyTypes.XSMALL_MEDIUM:
+        return `  
+            font-size: 11px;
+            font-weight: 500;
+            line-height: 16px;
+            `;
+      case TypographyTypes.XSMALL_REGULAR:
+        return `  
+            font-size: 11px;
+            font-weight: normal;
+            line-height: 16px;
+            `;
+      case TypographyTypes.BUTTON_MEDIUM_BIG:
+        return `  
+            font-size: 16px;
+            font-weight: 500;
+            line-height: 20px;
+            `;
+      case TypographyTypes.BUTTON_SMALL:
+        return `  
+            font-size: 14px;
+            font-weight: normal;
+            line-height: 14px;
+            `;
+    }
```

Comment:
add here a `default` case , if the type is not one of the following ( in the default case just think of the best default value- and return the most general type of typography).

Reply:
done